packages/tokens/src/bingel-dc/light-new.json

```diff
@@ -1549,25 +1549,25 @@
           "description": "Used as transparent layer on top of the muted, subtlest and subtle background to indicate the hover and pressed state."
         }
       }
-    }
-  },
-  "blanket": {
-    "plain": {
-      "value": "rgba({color.palette.blue.900}, {opacity.light.moderate})",
-      "type": "color",
-      "description": "Use for screen overlay that appears with dialogs or drawers."
-    }
-  },
-  "skeleton": {
-    "plain": {
-      "value": "{color.background.accent.grey.subtle}",
-      "type": "color",
-      "description": "Use for skeleton loading states."
     },
-    "subtle": {
-      "value": "{color.background.accent.grey.subtlest}",
-      "type": "color",
-      "description": "Use for the pulse or shimmer effect in skeleton loading states."
+    "blanket": {
```

Comment:
Can you please also add changeset for tokens update?

Reply:
Done